src/assets/scss/style.scss

```diff
@@ -1,202 +0,0 @@
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
-}
-
-
-.btn-primary {
-  background-color: $main-color;
-}
-.btn-outline-secondary {
-  color: #FFF;
-  border-color: $main-color;
-}
-a {
-  color: $main-color;
-}
-.margin-center {
-  margin: 0px auto;
-  text-align: center;;
-}
-.small-font {
-  font-size: .9rem;
-  line-height: .9rem;
-}
-
-header {
-  #navbarHeader, .navbar {
-    background-color: darken($main-color, 10%);
-
-    .text-description {
-      color: rgb(219, 211, 211);
-    }
-  }
-
-  .navbar-dark .navbar-toggler {
-    color: rgb(228, 228, 228);
-    border-color: rgb(228, 228, 228); 
-    box-shadow: none;
-  }
-
-  .list-link {
-    margin-top: 20px;
-    padding-left: 15px;
-    list-style-type: none;
-
-    li {
-      padding: 5px 0px;
-      @include transition;
-
-      a {
-        text-decoration: none;
-      }
-    }
-  }
-}
-
-main {
-  .search-bar {
-    margin: 40px 0px;
-    width: 100%;
-
-    .search-bar--input {
-      height: 60px;
-      border: 1px solid $main-color;
-    }
-  }
-  .box-form {
-    background-color: #FFF;
-    padding: 30px;
-    box-shadow: 0 0 30px rgba(214,215,216,.6);
-
-  }
-
-  .album-title {
-    margin: 0px auto;
-    text-align: center;
-    margin-bottom: 20px;
-    font-size: 2rem;
-    position: relative;
-    margin-bottom: 50px;
-    padding-bottom: 10px;
-
-    @include borderbotom;
-  }
-
-  .album {
-    .image-album {
-      width: 100%;
-      max-height: 400px;
-      overflow: hidden;
-
-      img {
-        object-fit: fill;
-        width: 100%;
-      }
-    }
-
-    .category-btn {
-      background-color: $main-color;
-      @include transition;
-
-      a {
-        color: #fff;
-        text-decoration: none;
-      }
-    }
-
-    .comments-list {
-      margin-top: 20px;
-      font-size: .9rem;
-
-      p {
-        margin: 0px;
-        padding: 0px;
-      }
-    }
-
-    .comments-section {
-
-      .comments-group {
-        border: 1px solid rgba(0,0,0,.125);
-
-        .comments-group--input {
-          border-width: 0px;
-          font-size: .9rem;
-        }
-        .comments-group--button {
-          font-size: .9rem;
-          border-width: 0px;
-          border-radius: 0px;
-          background-color: rgba(0,0,0,.125);
-          color: #524e4e;
-        }
-      }
-    }
-  }
-
-  .title-category {
-    position: relative;
-    margin-bottom: 30px;
-    padding: 10px;
-
-    @include borderbotom;
-  }
-  .category-list {
-    text-transform: capitalize;
-
-      .nav-link {
-        @include transition;
-      }
-  }
-}
-
-.spinner-border {
-  .visually-hidden {
-    margin: 0px auto;
-  }
-}
-
-footer {
-  .button-top {
-    background-color: $main-color;
-    color: #FFF;
-    font-size: 1rem;
-    border-radius: 50px;
-    padding: 5px 10px;
-    font-weight: 500;
-    border-width: 0px;
-    text-transform: lowercase;
-  }
-}
```

Comment:
It's good practice to have newline at end o file, see:  https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#:~:text=If%20content%20is%20added%20to,to%20include%20a%20newline%20character.

Also you can add some linting extensions for your IDE to check JS and styles code for consistent formatting and simple things like newlines for example.